grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -650,13 +650,14 @@ class ProductAvailabilityService {
         return data
     }
 
-    List getAvailableItems(Location location, List<Product> products) {
+    List getAvailableItems(Location location, List products, boolean excludeNegativeQuantity = false) {
+        // Flag to check if products is a list of products objects or list of products ids
+        boolean containsProductsIds = ListUtil.isTypeOfString(products)
```

Comment:
I tried to do a polymorphic method like:
```groovy
List getAvailableItems(Location location, List<String> products)
```
and leave the one:
```groovy
List getAvailableItems(Location location, List<Product> products)
```
but unfortunately it was shouting that there is already such a method, so it doesn't understand that `List<Product>` and `List<String>` is not the same.